pages/learn/develop/local-mode.md

```diff
@@ -223,14 +223,15 @@ Where `dockerhub-secret.yml` is a YAML file containing my private registry usern
 Sample secrets YAML file:
 
 ```yaml
-'https://index.docker.io/v2/':
+"https://index.docker.io/v2/":
   username: johnDoe
   password: myPassword
 ```
 
-[development]:/reference/OS/overview/2.x/#development-vs-production-images
-[cli]:/reference/cli
-[supervisor API]:/reference/supervisor/supervisor-api/
+[development]: /reference/OS/overview/2.x/#development-vs-production-images
+[cli]: /reference/cli
+[supervisor api]: /reference/supervisor/supervisor-api/
+
```

Comment:
Please don't remove spaces if you have tested that this doesn't break the anchor links.